pytest/lib/branches.py

```diff
@@ -57,7 +57,6 @@ def download(url: str) -> str:
 class Executables(typing.NamedTuple):
     root: pathlib.Path
```

Comment:
At this point we can also remove `root` and replace it with `root = property(lambda self: self.neard.parent)`. Not really sure if it’s worth the change though.